### PR TITLE
[DX-1437] organise --> organize (PR 1)

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/shared/dashboard-config.md
Comment (letzya) [inline tyk-docs/content/shared/dashboard-config.md:1]: @dcs3spp we need to open a pr for this change

Comment (letzya) [inline tyk-docs/content/shared/dashboard-config.md:1]: @Roeegg2 can you please remove this change as it's an auto-gen file from code so we'll change the source

File: tyk-docs/content/tyk-on-premises/bootstrapper-cli.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ To list the available flags, execute `tyk-analytics bootstrap -h`:
          --version              Show application version.
          --conf="tyk_analytics.conf"  
                                 Load a named configuration file.
-         --create-org           Create a new organisation.
-         --reuse-org=REUSE-ORG  Reuse the organisation with given ID.
-         --drop-org=DROP-ORG    Drop the organisation with given ID.
+         --create-org           Create a new organization.
```
Comment (letzya) [inline tyk-docs/content/tyk-on-premises/bootstrapper-cli.md]: This is output of a cli, we can't change it here. Please remove these changes

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md [hunk 2]
```diff
@@ -27,20 +27,20 @@ In this section, you’ll learn how to create a new organisation for your extern
 
 ## Step by step instructions
 
-1. From the **API Consumers > Organisations** menu, click **Add**.
+1. From the **API Consumers > Organizations** menu, click **Add**.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md:30]: cannot be changed as it's in the product (in the UI)

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md [hunk 3]
```diff
@@ -27,20 +27,20 @@ In this section, you’ll learn how to create a new organisation for your extern
...
 
-{{< img src="/img/dashboard/portal-management/enterprise-portal/organisations-menu.png" alt="Portal Organisations menu" >}}
-{{< img src="/img/dashboard/portal-management/enterprise-portal/add-org2.png" alt="Add a new Organisation" >}}
+{{< img src="/img/dashboard/portal-management/enterprise-portal/organisations-menu.png" alt="Portal Organizations menu" >}}
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md:32]: You can't change the path of existing URL

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md (hunk 2 above)
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md:30]:
```suggestion
1. From the **API Consumers > Organisations** menu, click **Add**.

**Note:** We are transitioning from GB to US notation. Some terms (e.g., UI, config files, APIs) still use GB notation. In this case, we use *organisation* with an *s* as appears in the source.
```

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations.md [hunk 4]
```diff
@@ -38,71 +38,71 @@ Please refer to the [email notification section]({{< ref "/product-stack/tyk-ent
...
-<br/>To proceed with the following steps, enable the Enable API consumers setting to register organisations.
+<br/>To proceed with the following steps, enable the Enable API consumers setting to register organizations.
 
 ## Step 1: Request org registration
 
 Register a developer account or use an existing one and log in to the developer portal as a developer.
-To start the organisation registration flow, click on the **Create an organisation** button in the top right corner of the screen.
-{{< img src="/img/dashboard/portal-management/enterprise-portal/become_an_organisation_navbar.png" alt="Become an organisation button" >}}
+To start the organization registration flow, click on the **Create an organization** button in the top right corner of the screen.
+{{< img src="/img/dashboard/portal-management/enterprise-portal/become_an_organisation_navbar.png" alt="Become an organization button" >}}
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations.md:53]: please fix the change in the url

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md (hunk 3 above)
Comment (Roeegg2) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md:32]: I only changed the alt tag, not the path. Should I revert the changes anyway?

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations.md (hunk 4 above)
Comment (Roeegg2) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations.md:53]: > I only changed the alt tag, not the path. Should I revert the changes anyway?

Same here

File: tyk-docs/content/shared/dashboard-config.md
Comment (dcs3spp) [reply tyk-docs/content/shared/dashboard-config.md:1]: https://github.com/TykTechnologies/tyk-analytics/pull/4072

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md (hunk 2 above)
Comment (letzya) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/add-organisations.md]: deleting this

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/customise-enterprise-portal/full-customisation/email-customization.md
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/customise-enterprise-portal/full-customisation/email-customization.md:1]: Please remove this file from the PR. We'll change it all together at a later stage when the product is ready

**Review by letzya — CHANGES_REQUESTED** [review]
Remove tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/customise-enterprise-portal/full-customisation/email-customization.md from the pr

**Review by letzya — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**